src/strings-in-block/class-attributes.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace WPML\PB\Gutenberg\StringsInBlock;
+
+class Attributes extends Base {
```

Comment:
@strategio (not blocking) I suggest adjusting the name and location of the new files:

- the path of the file must meet the namespace. In this case, it should be `src/WPML/PB/Gutenberg/StringsInBlock` (*)
- the filename must be the same as the class. In this case, `Attributes .php`
- In other words, the full path would be `src/WPML/PB/Gutenberg/StringsInBlock/Attributes .php` (*)

We can even decide that `src` represent our root namespace (`WPML`): as far as I've seen, it's a quite common approach.

You can even tell PHPStorm that the root namespace is under `src/WPML` (or `src`) so that new classes are created accordingly.
For instance, you can create a class `Foo` under the `WPML/Bar` namespace and PHPStorm will take care of creating it in the right place and with the right name.

Reply:
Honnestly, I am not convinced in this context @andreasciamanna. Maybe we should have used a namespace dedicated to this project making abstration of the parent project (i.e. `WPMLGutenberg`). But now I have another feature branch (reusable blocks) almost ready to be merged and it will use the long namespace with the current file structure.

Reply:
@kkarpieszuk I don't know which part of what I wrote does not convince you.

Also, I got lost after "abstraction"... we can discuss this by voice next week.

For now, I think we can survive with what we have, but we must better define (with the rest of the team and in written form) a convention for namespaces, naming and location of the files.

Or it will be soon become a nightmare.

Reply:
I agree @andreasciamanna. I can adjust the namespaces and file strucutre later when we decided.